src/lib/y2storage/disk_analyzer.rb

```diff
@@ -315,34 +315,18 @@ def mounted_filesystem?(filesystem)
 
     # Checks whether a device contains an installation repository
     #
-    # For all possible names of the given device, it is checked if any of that
-    # names is included in the URI of an installation repository (see
-    # {#repositories_devices}). Note that the names of all devices inside the
-    # given device are considered as names of the given device (see #{device_names}),
-    # (e.g., when a disk contains a partition being used as LVM PV, the names of the
-    # LVM LVs are considered as names of the disk).
+    # A device contains an installation repository if its kernel name or the kernel name of any of
```

Comment:
I find this documentation a bit confusing. It mentions several times the kernel names to later say that the URI can use any other kind of name. It looks to me like you are trying to describe more the internal implementation (converting everything to kernel names and then comparing) than the real effect.